runtime/src/main/java/uk/ac/manchester/tornado/runtime/common/RuntimeUtilities.java

```diff
@@ -58,6 +59,9 @@
     public static final int ONE_MEGABYTE = 1 * 1024 * 1024;
     public static final int ONE_KILOBYTE = 1 * 1024;
 
+    public static final String FPGA_OUTPUT_FILENAME = "output.log";
```

Comment:
Can these files be configured? We can put a name by default `outputFPGA.log` and `errorFPGA.log`. But we can also allow to obtain a name using the command line parameter. 

`-Dtornado.fpga.output.log=file.log`

`-Dtornado.fpga.error.log=fileError.log`

Reply:
I will change the default names, as it is suggested.

In my opinion, I do not see the point of having the name of the files being configurable by the user. The directory is configurable via the configuration file.

Reply:
M... true. I missed that. Ok, let's only update the names.